challenges/how-to-create-a-list.py

```diff
@@ -0,0 +1,31 @@
+# The challenge is to write a script that can generate all the fibonacci numbers form 1 to 100
```

Comment:
Can we separate these three solutions into three files, and place each of them in a folder titled "how-to-create-a-list"? I think that'll make this more consistent with the other challenge.

Reply:
So, you want the file structure to look like this?
./how-to-create-a-list
    /loop.py
    /recursion.py
    /list comprehensio.py

No big deal.

Reply:
Yes! That would be great. Thanks.